map/autoware_dynamic_lanelet_provider/config/dynamic_lanelet_provider.param.yaml

```diff
@@ -0,0 +1,5 @@
+/**:
+  ros__parameters:
+    dynamic_map_loading_grid_size: 50.0         # [m]
```

Comment:
this parameter is unused

Reply:
I removed this parameter from `config`, `source code` and `json schema`.